### DISCUSSION TITLE
How to append component to existing HoverEvent.ShowText?

### DISCUSSION THREAD

**Discussion by Kadeluxe** [KyoriPowered/adventure#502] [Q&A] [answered]
Given I have a `TextComponent` with some `HoverEvent.ShowText` already applied, is it possible to append another `TextComponent` to that hover component?

**Comment by kezz** [comment, marked as answer]
Yes, it's possible. You can obtain the hover event of a component, and, if the value of that hover event is a `Component`, you can manipulate this component as much as you would like before setting it as the hover event of the original component again.
